assets/js/modules/optimize/components/common/OptimizeIDFieldInstructions.js

```diff
@@ -37,13 +38,22 @@ export default function OptimizeIDFieldInstructions() {
 
 	return (
 		<p>
-			{ __(
-				'Please copy and paste your Optimize Container ID to complete your setup.',
-				'google-site-kit'
-			) }{ ' ' }
-			<Link href={ accountURL } external>
-				{ __( 'You can locate this here', 'google-site-kit' ) }
-			</Link>
+			{ createInterpolateElement(
+				__(
+					'Please copy and paste your Optimize Container ID to complete your setup.',
```

Comment:
```suggestion
					'Please copy and paste your Optimize Container ID to complete your setup. <LearnMoreLink />',
```

Or the same as for previous suggestions: `... <a>Learn more</a>`.

Reply:
@tofumatt, did you update this? Seems like this one has not been changed, rigght?

Reply:
Oh shoot, I missed this one. 😓 Made the change 👍🏻